### DISCUSSION TITLE
I am trying to deploy a small project using the unsplash API but when I open it is just a white screen

### DISCUSSION THREAD

**Discussion by leitel98** [community/community#48962] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Hello! I am trying to deploy a small project, a "pinterest clone" using unsplash as an API, but it wont deploy correctly, it is just a white page. Is there any advice you could give me to fix it? 

here is the page: https://leitel98.github.io/img-searcher/
here is the repository: https://github.com/leitel98/img-searcher

Maybe there is a syntax difference that I should be aware of or something, it deploys correctly on vscode

**Comment by mateusabelli** [comment]


**Reply by leitel98** [reply]
I kind of understand, can you tell me how to do it? I am not very experienced

**Reply by mateusabelli** [reply]
Hello there! I'm sorry but I was wrong. I forked your project and tried to reproduce the steps to see if I could find something but I got the same result as yours. Looking at the logs from GitHub Actions I've found something strange in the build process

<details>
<summary>Click to expand.</summary>

```
chmod: changing permissions of './_site': Operation not permitted
chmod: changing permissions of './_site/static': Operation not permitted
chmod: changing permissions of './_site/static/js': Operation not permitted
chmod: changing permissions of './_site/static/js/787.5b8e3c4f.chunk.js': Operation not permitted
chmod: changing permissions of './_site/static/js/main.6d1b[30](https://github.com/leitel98/img-searcher/actions/runs/4312419765/jobs/7529976401#step:5:32)6b.js.LICENSE.txt': Operation not permitted
chmod: changing permissions of './_site/static/js/787.5b8e3c4f.chunk.js.map': Operation not permitted
chmod: changing permissions of './_site/static/js/main.6d1b306b.js.map': Operation not permitted
chmod: changing permissions of './_site/static/js/main.6d1b306b.js': Operation not permitted
chmod: changing permissions of './_site/static/css': Operation not permitted
chmod: changing permissions of './_site/static/css/main.0647fbd8.css': Operation not permitted
chmod: changing permissions of './_site/static/css/main.0647fbd8.css.map': Operation not permitted
chmod: changing permissions of './_site/favicon.ico': Operation not permitted
chmod: changing permissions of './_site/manifest.json': Operation not permitted
chmod: changing permissions of './_site/logo192.png': Operation not permitted
chmod: changing permissions of './_site/robots.txt': Operation not permitted
chmod: changing permissions of './_site/assets': Operation not permitted
chmod: changing permissions of './_site/assets/css': Operation not permitted
chmod: changing permissions of './_site/assets/css/style.css': Operation not permitted
chmod: changing permissions of './_site/index.html': Operation not permitted
chmod: changing permissions of './_site/asset-manifest.json': Operation not permitted
chmod: changing permissions of './_site/logo512.png': Operation not permitted
```

</details>

While I ran the project in my local environment everything worked fine. I believe this strange output from GH Actions could be the reason why the deployment has failed, but I can't tell you for sure. I'd recommend you to try to host this project in a different provider like Vercel or Netlify.

**Reply by leitel98** [reply]
Wow, thank you for noticing it! I presume it is a chrome error of some kind maybe?, I don't know what is chmod really, I will try to open it on another browser and see if it works, if not I will follow your advice, thanks!

**Comment by Anmol-Baranwal** [comment]
GitHub Pages is basically for the frontend. 
So, if you want to host a backend project, use [render](https://render.com/), otherwise you can use [vercel](https://vercel.com/dashboard)
